flo-workflow/src/main/java/com/spotify/flo/context/MemoizingContext.java

```diff
@@ -215,10 +215,12 @@ private void mapMemoizer(Memoizer<?> memoizer) {
     final Task<T> task = evalBundle.task;
     final Memoizer<T> memoizer = evalBundle.memoizer;
 
-    final Value<T> tValue = delegate.invokeProcessFn(taskId, processFn);
-    tValue.consume(v -> memoizer.store(task, v));
+    final Value<T> newValue = processFn.get().map(v -> {
```

Comment:
Won’t this call to `processFn.get()` change when it’s called? From being decided by the delegate to being called here. I would best this code inside a `() -> ...` block so it can be passed into the delegate directly.

Reply:
Yeah, good point.